lib/utils.js

```diff
@@ -115,6 +115,25 @@ var Utils = module.exports = {
       options.where = Utils.mapWhereFieldNames(options.where, Model);
     }
 
+    if (Array.isArray(options.order)) {
+      options.order.forEach(function(oGroup) {
+        var OrderModel, attr;
+        if (Array.isArray(oGroup)) {
+          OrderModel = Model;
+          attr = oGroup[oGroup.length - 2];
```

Comment:
Ok thanks. I will add that case. I didn't see that example in the docs here http://docs.sequelizejs.com/en/latest/docs/querying/#ordering or here http://docs.sequelizejs.com/en/latest/api/model/#findall .  I now see that is possible in the code